pkg/lib/machines.js

```diff
@@ -570,6 +573,30 @@
                     });
             }
 
+            /* Try to get change notifications via the internal
+               /packages D-Bus interface of the bridge.  Not all
+               bridges support this API, so we still get the first
+               version of the manifests via HTTP in request_manifest.
+            */
+
+            function watch_manifests() {
+                var dbus = cockpit.dbus(null, { bus: "internal",
```

Comment:
Good point.  Not all bridges support the /packages API but I guess we can still use it to make first contact.  I'll do this as a separate commit.